### PR TITLE
Three.js r141

### PR DESCRIPTION
**Pull request by takahirox**
For Three.js upgrade to r141

### CODE REVIEW

File: packages/three-particle-emitter/lib/cjs/index.js [hunk 1]
```diff
@@ -98,6 +115,7 @@ class ParticleEmitter extends three_1.Mesh {
             fragmentShader,
             transparent: true,
             depthWrite: false,
+            // @ts-ignore
```
Comment (takahirox) [inline packages/three-particle-emitter/lib/cjs/index.js:120]: TODO: Check if this workaround is ok

File: packages/three-particle-emitter/src/index.ts [hunk 2]
```diff
@@ -151,7 +151,8 @@ export class ParticleEmitter extends Mesh {
       fragmentShader,
       transparent: true,
       depthWrite: false,
-      fog: true,
+      // @ts-ignore
```
Comment (netpro2k) [inline packages/three-particle-emitter/src/index.ts:156]: Hmm yeah to fix I think you would need to create a ShaderMaterial subclass or just override the type here. This seems fine.

File: packages/easing-functions/lib/cjs/index.js [hunk 3]
```diff
@@ -6,6 +6,7 @@
 // http://robertpenner.com/easing/
 // http://robertpenner.com/easing_terms_of_use.html
 Object.defineProperty(exports, "__esModule", { value: true });
+exports.bounceInOut = exports.bounceOut = exports.bounceIn = exports.backInOut = exports.backOut = exports.backIn = exports.elasticInOut = exports.elasticOut = exports.elasticIn = exports.circularInOut = exports.circularOut = exports.circularIn = exports.exponentialInOut = exports.exponentialOut = exports.exponentialIn = exports.sinusoidalInOut = exports.sinusoidalOut = exports.sinusoidalIn = exports.quinticInOut = exports.quinticOut = exports.quinticIn = exports.quarticInOut = exports.quarticOut = exports.quarticIn = exports.cubicInOut = exports.cubicOut = exports.cubicIn = exports.quadraticInOut = exports.quadraticOut = exports.quadraticIn = exports.linear = void 0;
```
Comment (netpro2k) [inline packages/easing-functions/lib/cjs/index.js:9]: I wonder why these other changes happened. Different typescript compiler versions I guess.

**Review by netpro2k — APPROVED** [review]
LGTM. The changes in the compiled files are slightly unexpected but probably just due to the typescript version upgrade.

File: packages/three-particle-emitter/src/index.ts (hunk 2 above)
Comment (takahirox) [reply packages/three-particle-emitter/src/index.ts:156]: Added TODO to resolve the root issue properly later so far.

**Pull request merged**